web/docs/_static/openapi.yaml

```diff
@@ -121,6 +121,19 @@ paths:
   /change_password:
     post:
       summary: Change the current user's password.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                old_password:
+                  type: string
+                  example: SuperSecretPassword
+                new_password:
+                  type: string
+                  example: NewSuperSecretPassword
```

Comment:
This has been removed from the implementation, if I remember correctly

Reply:
/reset_password requestBody no longer includes password and password2, but that is PR #58 and /change_password definitely needs the user to confirm current password.